src/Authentication/Program.cs

```diff
@@ -296,7 +296,8 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
     services.AddHealthChecks().AddCheck<HealthCheck>("authentication_health_check");
 
     services.AddSingleton(config);
-    services.Configure<GeneralSettings>(config.GetSection("GeneralSettings"));     
+    services.Configure<GeneralSettings>(config.GetSection("GeneralSettings"));
+    services.Configure<PaginationOptions>(config.GetSection("PaginationSize"));
```

Comment:
PaginationOptions

Reply:
ok

Reply:
this isnt resolved GetSection("PaginationSize") must be updated to GetSection("PaginationOptions")

Reply:
ok, now its fixed in program.cs too